spec/unit/manifest_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'etc'
+
+module ThreeScale
+  module Backend
+    describe Manifest do
+      describe '.thread_safe?' do
+        it 'returns false' do
+          expect(described_class.thread_safe?).to eq(false)
+        end
+
+      end
+
+      describe '.compute_workers' do
+        it 'returns PUMA_WORKERS_CPUMULT * <number of CPUs> if no PUMA_WORKERS environment variable is set' do
+          stub_const('ENV', ENV.to_h.merge('PUMA_WORKERS' => nil))
+          expect(described_class.compute_workers(Etc.nprocessors))
```

Comment:
sorry I missed this the first time - `Etc.nprocessors` is an implementation detail. You have the number of CPUs actually offered by a method in the `Manifest` module, so use that (also use a `let` assignment and reuse in other examples).

Reply:
(the number of CPUs actually comes from a helper in `ThreeScale::Backend::Util`)